parl/remote/remote_wrapper.py

```diff
@@ -0,0 +1,237 @@
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cloudpickle
+import inspect
+import threading
+import zmq
+
+from parl.utils import logger, to_str, to_byte
+from parl.utils.communication import loads_argument, loads_return,\
+    dumps_argument, dumps_return
+from parl.remote.client import get_global_client
+from parl.remote import remote_constants
+from parl.remote.utils import locate_remote_file
+from parl.remote.exceptions import RemoteError, RemoteAttributeError,\
+    RemoteDeserializeError, RemoteSerializeError, ResourceError, FutureFunctionError
+
+
+class RemoteWrapper(object):
+    """
+    Wrapper for remote class in client side.
+    1. request cpu resource and submit job to the master
+    2. handle the function calling of user, mapping it to the remote procedure call through zmq
```

Comment:
return the result of the function called by the user, using the remote computation resource.

Reply:
done